src/Checkboxes/Checkboxes.jsx

```diff
@@ -23,23 +24,44 @@ const Checkboxes = ({
 }) => {
   const classes = classBuilder(classBlock, classModifiers, className);
 
+  const [selection, setSelection] = useState(value ? value : []);
+
+  const updateSelection = ({ target }) => {
+    if (target.checked) {
+      setSelection((oldSelection) => [...oldSelection, target.value]);
+    } else {
+      setSelection(selection.filter((s) => s !== target.value));
```

Comment:
```suggestion
      setSelection(oldSelection => oldSelection.filter((s) => s !== target.value));
```

(Personally, I'd use the variable name `prev` rather than `oldSelection`... but I was being consistent with line 31.)

Reply:
Swapped to prev for both